### PR TITLE
add field for virtual network gateway and virtual network gateway connection

### PR DESCRIPTION
**Pull request by chenjianfei2017**

### CODE REVIEW

File: website/docs/d/virtual_network_gateway.html.markdown [hunk 1]
```diff
@@ -43,6 +43,8 @@ output "virtual_network_gateway_id" {
 
 * `active_active` - Is this an Active-Active Gateway?
 
+* `private_ip_address_enabled` - Whether private IP needs to be enabled on this gateway for connections or not.
```
Comment (katbyte) [inline website/docs/d/virtual_network_gateway.html.markdown:46]: grammer?
```suggestion
* `private_ip_address_enabled` - Whether a private IP will be used for this  gateway for connections.
```

or if the description is right
```suggestion
* `private_ip_address_required` - Whether private IP needs to be enabled on this gateway for connections or not.
```

**Review by katbyte — CHANGES_REQUESTED** [review]
i've left some comments inline to address

**Comment by chenjianfei2017** [comment]
@katbyte  Thanks for your review! I changed the code according to your comments. pls help to review this pr.

**Review by katbyte — APPROVED** [review]
Thanks, LGTM now 👍

**Comment by chenjianfei2017** [comment]
@katbyte Could you pls help to merge this pr, I have no permission to merge, thx.

**Comment by katbyte** [comment]
Was waiting for tests to pass, they have now so merging!

**Pull request merged**